apps/frontend/src/app/2_molecules/BalanceInput/index.tsx

```diff
@@ -0,0 +1 @@
+export * from './BalanceInput';
```

Comment:
This file is probably not needed as we won't use it outside of the fronted dapp, right?